certified-connectors/Mobsim Send SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,150 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MOBSIM Send SMS",
+    "description": "With Mobsim Send SMS, send text messages: alerts, notifications, marketing messages and more.",
+    "version": "1.0",
+    "contact": {
+      "name": "Mobsim Support",
+      "url": "https://www.mobsim.com.br/",
+      "email": "contato@mobsim.com.br"
+      }
+  },
+  "host": "mobsim-api.com.br",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/sms": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
```

Comment:
@jpjsantos please provide a schema for your 200 response if this is an action, not a trigger.